deep-learning/src/main/scala/com/microsoft/azure/synapse/ml/onnx/ImageFeaturizer.txt

```diff
@@ -0,0 +1,16 @@
+The ``ImageFeaturizer`` relies on a ``ONNXModel`` to do the
+featurization of the image(s).  One can set this model directly using the
+onnxModel parameter, or use setModel to set a known model name that will
```

Comment:
these text files are no longer part of our build. Perhaps we can just incorporate into scala docstring if needed

Reply:
ok, I have a separate task to cleanup related docs and add examples, so I'll do that as part of that task.